odxtools/basicstructure.py

```diff
@@ -350,8 +359,8 @@ def param_sort_key(param: Parameter) -> Tuple[int, int]:
                     # END-OF-PDU fields do not exhibit a fixed bit
                     # length, so they need special treatment here
                     dct = None
-                    if hasattr(params[param_idx], "dop"):
-                        dop = params[param_idx].dop  # type: ignore
+                    if hasattr(params[param_idx], "_dop"):
+                        dop = params[param_idx]._dop  # type: ignore[attr-defined]
```

Comment:
Why?

Reply:
it is slightly faster because `.dop` requires a function call...

Reply:
correction: I have a dataset where the DOP cannot be resolved (in this case, because the dataset is faulty referencing a non-existant object), and for this I'll get an exception in `.dop` while I just want it to be ignored (for now)...

Reply:
disable strict mode?